skt/runner.py

```diff
@@ -145,14 +145,14 @@ def getresultstree(self, jobid, logs=False):
 
     def __forget_cid(self, cid):
         """
-        Remove a job or recipe from the current list
+        Remove a job or recipe set from self.job_to_recipe_map.
```

Comment:
I would prefer a human-readable name of what's in `self.job_to_recipe_map` used here, and also as a comment over it in `__init__()`. E.g. something like "the job->recipe IDs map". I won't hold my ACK over this, though.